site/lists/resources.scroll

```diff
@@ -0,0 +1,12 @@
+title Programming languages ebooks
+import settings.scroll
+columnWidth 100
+
+* List of programming languages ebooks
+
+groups index
+endSnippet
+
+import resourcesData.scroll
```

Comment:
great idea to separate the data from the page. i would just rename the file to `ebooksData.scroll`